### DISCUSSION TITLE
IP Address error in Netbox 3.6.0

### DISCUSSION THREAD

**Discussion by niask11** [netbox-community/netbox#13631] [Help Wanted!] [answered]
Hello. I've upgraded from 3.5.7 to 3.6.0, the upgrade process ended up without errors, but now I've got the following issue: whenever I click on an IP assigned to any VM I've got this:
![изображение](https://github.com/netbox-community/netbox/assets/44463031/9adf8e05-0108-48e2-8f31-7f912d001909)
If I go to the VM I see the required IP on an interface, as primary etc, but when I click on it - always this error. BTW prefixes work fine.
Now I cannot browser all IPs. Any help will be appreciated.

**Comment by jeremystretch** [comment, marked as answer]
Please see #13619
